Cargo.lock

```diff
@@ -2162,9 +2188,9 @@ checksum = "ff011a302c396a5197692431fc1948019154afc178baf7d8e37367442a4601cf"
 
 [[package]]
 name = "openssl-src"
-version = "111.25.3+1.1.1t"
+version = "111.25.2+1.1.1t"
```

Comment:
I checked the PR and it looks like it edits `Cargo.lock` directly. When rebasing this PR, I had merge conflicts in `Cargo.toml`, and under the assumption that `Cargo.lock` is fully generated, I just delete them and have them regenerated.
Maybe a way to set the version 'more officially' would be to create a dependency to the crate in `Cargo.toml` with the desired version.

I fear that editing the `Cargo.lock` again will lead to similar issues further down the road, and I will wait for your feedback.

Reply:
`cargo update -p openssl-src --precise '111.25.3+1.1.1t'` could fix that. I'll find a time bumping some direct dependencies to have a longer-term fix.

Reply:
That will work, and thanks for the command-line, too. Otherwise I would have found myself trying to edit the `Cargo.lock` file 😅.

Implemented in 99316c8e4356444bec99f5cb4d9900ff1e8e8294.